osi_object.proto

```diff
@@ -61,6 +61,22 @@ message StationaryObject
         //
         optional Color color = 4;
 
+        // The IDs of the lanes that the object is assigned to.
+        //
+        // \note Might be multiple if the object streches over multiple lanes.
+        //
+        // \note OSI uses singular instead of plural for repeated field names.
```

Comment:
I see this note repeated over and over again across all files. A quick search yields 70 hits for this exact phrase.

I do not know the original reason for including the note, but I suggest to just remove it. In my opinion, it is stating the obvious. The field is a repeated field and its name uses in fact singular. Furthermore, the above text makes it abundantly clear that there may be multiple IDs. 

However, I would also suggest to add this information to the [interface-conventions topic](https://github.com/OpenSimulationInterface/osi-documentation/blob/migration/contributing/naming_conventions.adoc) in the general documentation.

Reply:
created issues to handle this topic.